lib/elsa/supervisor.ex

```diff
@@ -72,13 +74,32 @@ defmodule Elsa.Supervisor do
   * `:config` - Optional. Consumer configuration options passed to `brod_consumer`.
 
 
+  ## Consumer Config
+
+  * `:topic` - Required. Topic to subscribe to.
+
+  * `:partition` - Required. Topic partition to subscribe to.
+
+  * `:begin_offset` - Required. Where to begin consuming from the topic. Must be either `:earliest` or `:latest`.
```

Comment:
This can also just be a specified offset, the user can track the offset themselves.

Reply:
Yeah makes sense.